discord/ext/menus/__init__.py

```diff
@@ -638,10 +642,25 @@ async def update(self, payload):
                         await button(self, payload)
             else:
                 await button(self, payload)
-        except Exception:
-            # TODO: logging?
-            import traceback
-            traceback.print_exc()
+        except Exception as exc:
+            await self.menu_update_error_handler(exc)
+
+    async def menu_update_error_handler(self, exc: Exception):
```

Comment:
```suggestion
    async def on_menu_button_error(self, exc):
```

(there's no type hinting in the library at the moment)